src/Integration/MefServices/ActiveSolutionTracker.cs

```diff
@@ -24,6 +24,8 @@
 using Microsoft.VisualStudio;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.Interop;
+using SonarLint.VisualStudio.Core;
+using ErrorHandler = Microsoft.VisualStudio.ErrorHandler;
```

Comment:
Changed due to Core dependency, although we probably want to use Core's ErrorHandler, no?

Reply:
Unfortunately, we're not consistent in which one we use (although Core is supposed to work the same way as the VS version). In fact, we're so inconsistent it probably doesn't matter which one we pick.

However, we could try to establish a rule of thumb to follow: if the assembly reference VS assemblies, use the VS one; otherwise use Core.
It's a bit arbitrary, but at least it would be consistent. What do you think?

Reply:
So basically use the VS one everywhere, except in Core? 🤔 Can do, and then make it internal.